rfcs/interop_2022.md

```diff
@@ -0,0 +1,127 @@
+
+### Text
+
+Proposal: [interop-2022/#13](https://github.com/web-platform-tests/interop-2022/issues/13)  
+Tests: [`interop-2022-text`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2022-text)
+
+### New Viewport Units
+
+Proposal: [interop-2022/#4](https://github.com/web-platform-tests/interop-2022/issues/4)  
+Tests: `interop-2022-viewport` (tests forthcoming)
+
+See also the [Viewport Investigation project](https://github.com/web-platform-tests/interop-2022/issues/41) which is part of the Interop 2022 effort, but not part of the metric.
+
+### Web Compat
+
+Proposal: [interop-2022/#9](https://github.com/web-platform-tests/interop-2022/issues/9)  
+Tests: [`interop-2022-webcompat`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2022-webcompat)
+
+### Interop 2021
+
+The 5 focus areas from Interop 2021 are carried forward and included in the metric. See [interop-2022/#2](https://github.com/web-platform-tests/interop-2022/issues/2) for discussion.
+
+- [`interop-2021-aspect-ratio`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-aspect-ratio)
+- [`interop-2021-flexbox`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-flexbox)
+- [`interop-2021-grid`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-grid)
+- [`interop-2021-position-sticky`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-position-sticky)
+- [`interop-2021-transforms`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-transforms)
+
+[Query for `interop-2021-*`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-aspect-ratio%20or%20label%3Ainterop-2021-flexbox%20or%20label%3Ainterop-2021-grid%20or%20label%3Ainterop-2021-position-sticky%20or%20label%3Ainterop-2021-transforms)
+
+### Updating the tests
+
+No test suite is perfect, and there will likely be a need to add, remove or rewrite tests during the course of the year. To suggest such non-trivial changes that would benefit from review, file an issue on the [interop-2022 repo](https://github.com/web-platform-tests/interop-2022). The [interop-2022 team](https://github.com/orgs/web-platform-tests/teams/interop-2022) will discuss at least monthly, and make decisions based on consensus. Anyone who would like to contribute to the Interop 2022 effort is welcome to join this team.
```

Comment:
For the final sentence I suggest instead: "The core team will nominate the initial partcipants to the Interop-2022 team; thereafter the team will be able to set its own criteria for membership and participation, subject to the wpt code of conduct."

Practically I think the effect will be similar, but I think it makes sense to explicitly delegate responibility for team membership to the team itself, rather than either making it a free-for-all (which assumes no possibility of bad actors), or requiring decisions to go through the core team.

Reply:
Oops, I never updated according to this suggestion. Now done!